supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -2,6 +2,14 @@
 [[general-formatting]]
 = General formatting
 
+[[date-formats]]
+== Date formats
+
+Follow _The IBM Style Guide_  guidance of day Month year for date formats, for example 3 October 2019.
+
+When the format, day Month year, causes a presentation or clarity issue, use instead: Month day, year (for example, October 3, 2019).
```

Comment:
The first set of commas should be removed: When the format day Month year causes...